CoreFoundation/String.subproj/CFString.c

```diff
@@ -491,6 +489,7 @@ CF_INLINE Boolean __CFStrEncodingCanBeStoredInEightBit(CFStringEncoding encoding
         case kCFStringEncodingInvalidId:
         case kCFStringEncodingUnicode:
         case kCFStringEncodingNonLossyASCII:
+	default:
```

Comment:
Moving `default` up is a very strange coding style. I don't think we should make this change.